openwisp_controller/connection/tests/test_models.py

```diff
@@ -919,7 +919,7 @@ def _assert_applying_conf_test_command(mocked_exec):
             self.assertEqual(conf.status, 'applied')
 
         with self.subTest('openwisp_config < 0.6.0a: exit_code 0'):
-            conf.config = '{"interfaces": []}'
+            conf.config = '{"interfaces": [{"name": "eth00","type": "ethernet"}]}'
```

Comment:
Empty fields are omitted when generating configuration. Hence, the checksum remained same throughout the test which led to a failing test case. I have updated the test case to avoid this issue.